hs_file_types/models/geofeature.py

```diff
@@ -85,11 +85,11 @@ def get_html_forms(self, datatset_name_form=True):
         root_div = div("{% load crispy_forms_tags %}")
         with root_div:
             super(GeoFeatureFileMetaData, self).get_html_forms()
-            with div(cls="col-lg-6 col-xs-12"):
+            with div(cls="content-block"):
                 div("{% crispy geometry_information_form %}")
-            with div(cls="col-lg-6 col-xs-12 col-md-pull-6", style="margin-top:40px;"):
+            with div(cls="content-block"):
                 div("{% crispy spatial_coverage_form %}")
-            with div(cls="col-lg-6 col-xs-12"):
```

Comment:
I like that you're removing styling from the html building.  @engineSound - Could you review the templates, dominate and changes to organizing styling?  Also, what are your thoughts on dominate vs templates?